### PR TITLE
Add a Sliceviewer page to the user documentation

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

The Sliceviewer documentation was still on the wiki. This creates a page for the new Sliceviewer under the workbench directory in the docs and changes all of the references internally to this page.

**To test:**

* Build the docs
* Review the new page

*There is no associated issue.*

*This does not require release notes* because **it is not a new feature**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by RichardWaiteSTFC** [comment]
Should this have the milestone 6.1?

**Review by RichardWaiteSTFC — CHANGES_REQUESTED** [review]
Looks good, informative screenshots and all the links to the sliceviewer docs work. Just one thing I would add about the cursor information widget.

File: docs/source/workbench/sliceviewer.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+presented in the single-pixel line plots mode.
+
+.. _sliceviewer_cursor:
+
+Cursor Information Widget
+-------------------------
+
+The revamped Sliceviewer has merged several features from the SpectrumViewer
+in MantidPlot. One of these new features is the ability to show information
+regarding a given pixel as the mouse cursor moves of the image. The new table
```
[the remark this answers is not in the document]
Comment (martyngigg) [reply docs/source/workbench/sliceviewer.rst:102]: Good call. I've just updated the page.

**Comment by martyngigg** [comment]
@RichardWaiteSTFC I don't suppose you'd have time to review this today?

**Comment by RichardWaiteSTFC** [comment]
> @RichardWaiteSTFC I don't suppose you'd have time to review this today?

@martyngigg Will do, sorry I forgot about this!

File: docs/source/workbench/sliceviewer.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+   :width: 75%
+   :align: center
+
+
+Non-Orthogonal Axes View
+------------------------
+
+If the units of an MDWorkspace are HKL and the workspace has an attached
+UB matrix then the option to display the image using axes aligned with the
+crystal (so-called non-orthogonal view) will be now be visible:
```
Comment (RichardWaiteSTFC) [inline docs/source/workbench/sliceviewer.rst:58]: Such a minor nit-pick, but I don't think this is technically true. If a workspace fulfils your criteria (i.e. has HKL units and a UB present)  but the angles are set to 90 (within some tolerance?) then the button is not enabled (as you would expect!). However I believe the meaning is clear and I think this should PR should be merged before smoke testing so I am approving (I'm just sorry I didn't pick up on it before).

**Review by RichardWaiteSTFC — APPROVED** [review]

**Comment by rosswhitfield** [comment]
@martyngigg Which data set did you use to generate the figures for the peaks overlay part of the sliceviewer page. I would like to update the page as part of #31282

**Comment by martyngigg** [comment]
@rosswhitfield I used the SXD23767 data from the [training data](http://sourceforge.net/projects/mantid/files/Sample%20Data/TrainingCourseData.zip/download) and this script

```python
# import mantid algorithms, numpy and matplotlib
from mantid.simpleapi import *
import matplotlib.pyplot as plt
import numpy as np

SXD23767 = Load(Filename='SXD23767.raw', LoadMonitors='Exclude')
peaksws = Load('peaks_qLab.nxs')
FindUBUsingFFT(PeaksWorkspace='peaksws', MinD=0.8, MaxD=10)
IndexPeaks(peaksws, Tolerance=0.05)
CopySample(InputWorkspace='peaksws', OutputWorkspace='SXD23767',
           CopyName=False, CopyMaterial=False, CopyEnvironment=False, CopyShape=False)

mdws = ConvertToDiffractionMDWorkspace(SXD23767,
                                       OutputDimensions='HKL',
                                       OneEventPerBin=False,
                                       LorentzCorrection=True)
                                    
IntegratePeaksMD(
    InputWorkspace=mdws,
    PeaksWorkspace=peaksws,
    PeakRadius=0.5,
    BackgroundInnerRadius=1.1,
    BackgroundOuterRadius=1.4,
    OutputWorkspace='peaksws_sphere_withbkgd')

peaksws_sphere_withbkgd = mtd['peaksws_sphere_withbkgd']
while peaksws_sphere_withbkgd.getNumberPeaks() > 25:
    DeleteTableRows(peaksws_sphere_withbkgd, Rows=peaksws_sphere_withbkgd.getNumberPeaks() - 1)
```

I think I must have played a bit to remote some peaks so there weren't so many found.

**Comment by rosswhitfield** [comment]
Thank you!

**Pull request merged**